app/web/features/search/FilterDialog.tsx

```diff
@@ -213,15 +240,26 @@ export default function FilterDialog({
               />
 
               <Select
-                id="can_host_status_filter"
+                id="can_host_status_filter_1"
+                variant="outlined"
+                label={"Hosting status"}
```

Comment:
upsi, just realized this hardcoded label

Reply:
Ah yeah probably want a translation there instead. Looks like there are already some if they apply here if you search "Hosting status" in VSCode.

Reply:
solved ✅